client/src/screens/groups.screen.js

```diff
@@ -145,20 +155,27 @@ class Groups extends Component {
 
   goToGroup(group) {
     const { navigate } = this.props.navigation;
-    navigate('Group', { groupId: group.id, title: group.name });
+    const tags = (group.tags.map((elem) => {return '#'+elem.name})).join(',')
+
+    navigate('Group', { groupId: group.id, title: group.name, tags: tags });
```

Comment:
I don't think you want to put tags in the navigation params - the groupId and title should be enough. You can get the rest from within the components on that screen.

Reply:
Why refetch data we already know?